pgdrive/scene_creator/vehicle_module/routing_localization.py

```diff
@@ -236,25 +241,46 @@ def set_force_calculate_lane_index(self, force: bool):
     def __del__(self):
         logging.debug("{} is destroyed".format(self.__class__.__name__))
 
-    def get_current_lateral_range(self) -> float:
+    def get_current_lateral_range(self, current_position, pg_world) -> float:
```

Comment:
RoutingLocalizationModule instance already has a `self.pg_world`, why we need to pass in the pg_world argument in this function?

Reply:
No, pg_world doesn't exist in routing_localization module